templates/app/controllers/store_controller.rb

```diff
@@ -3,7 +3,7 @@
 class StoreController < Spree::BaseController
   include Spree::Core::ControllerHelpers::Pricing
   include Spree::Core::ControllerHelpers::Order
-  include SolidusStarterFrontend::Taxonomies
+  include Taxonomies
```

Comment:
Do we actually need the `#taxonomies` helper in the devise controllers? can we just inline the method where it's used?

Maybe out of the scope, but give this is no more library code, but rather a reference implementation I think we can be more specific and less "future-proof", thoughts?

Reply:
I'll check if Taxonomies is used in the Devise controllers. If not, we can inline the module in `Spree::BaseController`. TODO

Reply:
@elia I found that the Devise controllers call the `taxonomies` helper as follows:

* Devise controllers load the storefront layout.
* The storefront layout renders the `header` partial.
* The `header` calls `taxonomies`.

Viewed from a Devise controller page, the `taxonomies` helper is called in the headers section, highlighted in red below:

![image](https://user-images.githubusercontent.com/61476/190115500-2af8d629-46b8-4e1f-9288-d31eb01e2721.png)